node/core/prospective-parachains/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "polkadot-node-core-prospective-parachains"
+version = "0.9.16"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+futures = "0.3.19"
+gum = { package = "tracing-gum", path = "../../gum" }
+parity-scale-codec = "2"
+thiserror = "1.0.30"
+fatality = "0.0.6"
+bitvec = "1"
+
+polkadot-primitives = { path = "../../../primitives" }
+polkadot-node-primitives = { path = "../../primitives" }
+polkadot-node-subsystem = { path = "../../subsystem" }
+polkadot-node-subsystem-util = { path = "../../subsystem-util" }
+
+[dev-dependencies]
+assert_matches = "1"
+polkadot-node-subsystem-test-helpers = { path = "../../subsystem-test-helpers" }
+polkadot-node-subsystem-types = { path = "../../subsystem-types" }
+polkadot-primitives-test-helpers = { path = "../../../primitives/test-helpers" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-keystore = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-keyring = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-keystore = { git = "https://github.com/paritytech/substrate", branch = "master" }
+
+[features]
+# If not enabled, the dispute coordinator will do nothing.
+disputes = []
```

Comment:
Why do we have this for this subsystem? It appears unused.

Reply:
Copypasta I think, @BradleyOlson64

Reply:
Added this to my docs edits pr